app/src/main/res/layout/topic_index_items.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:id="@+id/cardView"
+    android:layout_width="match_parent"
+    android:layout_height="230dp"
+    android:layout_margin="5dp"
+    app:cardCornerRadius="3dp">
+
+<LinearLayout
```

Comment:
Alignment of LinearLayout is not correct.

Also, make sure you press `option+command+L` regularly to reformat the code automatically.